### DISCUSSION TITLE
Vertical bar/dark mode - upower icon question

### DISCUSSION THREAD

**Discussion by crawfordlong** [JakeStanger/ironbar#790] [Q&A]
I am very, VERY new to this bar (so apologies in advance if I miss something obvious), but I like the ease of use so far. Two quick questions about the upower module and a closely related question about the clock module:

1. My bar is oriented vertically with a width of 42px. I'm not wedded to this specific width, but that's where I am right now. The width of the upower module is quite a bit greater than that, making the bar much wider than I'd like. **Is there a way to add a carriage return after the battery icon or otherwise orient the module more optimally for a vertical bar?**
2. My bar has a dark background. I may fix the issue I'm having in CSS (e.g., by making the bar more transparent against my light wallpaper) but, unlike the audio and clipboard modules, the battery icon is a dark green (I think?) color that does not show up against my dark background. Is there a "dark mode" I'm missing? 
3. In the clock module, I have the time displaying as %H\r%M. Anyone have a more elegant vertical bar display for time?

**Comment by JakeStanger** [comment]
Hey, I can answer the first two:

1. You should be able to do that by adding a newline to the `format` option. I've also just pushed a fix as the module was ignoring the bar orientation, meaning it should just work for you on the latest commit now.
2. The UPower module is currently lacking the ability to customise the icons on-par with other modules, which means it's limited to using icons from the configured icon theme. You will need to tell Ironbar to use a suitable GTK icon theme for dark backgrounds.
